### ISSUE TITLE
Many videos will not play, or stop during play.

### ISSUE THREAD

**Issue by kwoodall** [yuliskov/SmartTube#2846]
The video seems to be starting, but freezes with the spinner rotating, then finally times out.

**Comment by kennethpangky** [comment]
Yup, experiencing the same on multiple devices. Have tried changing codec or resolution - those that won't play just won't properly. Might play a short snippet but stops soon.

Same videos plays fine on the original YT app, if that helps.

**Comment by kwoodall** [comment]
Here are a few I noted with this issue last night:

https://www.youtube.com/watch?v=oBbT7WRlDwQ&list=FL0a6qruXrEVcQV0ezREYSEg&index=1&t=40s&pp=gAQB
https://www.youtube.com/watch?v=Q84Bk0XXiJE&list=FL0a6qruXrEVcQV0ezREYSEg&index=2&t=27s&pp=gAQB
https://www.youtube.com/watch?v=vJykr1MQZ0U&list=FL0a6qruXrEVcQV0ezREYSEg&index=3&t=20s&pp=gAQB

These played fine in the YT player and I tried both 18.92 and 19.05 beta
where they don't work.
